rmgpy/cantherm/statmech.py

```diff
@@ -391,7 +391,7 @@ def load(self):
         if E0_withZPE is None:
             # The E0 that is read from the log file is without the ZPE and corresponds to E_elec
             if E0 is None:
-                E0 = energyLog.loadEnergy(self.frequencyScaleFactor)
+                E0 = energyLog.loadEnergy(self.frequencyScaleFactor, self.modelChemistry)
```

Comment:
This will crash for non-Molpro log files.

Reply:
I also added it in gaussian.py and qchem.py